client/components/theme/index.jsx

```diff
@@ -172,8 +176,12 @@ const Theme = React.createClass( {
 								context: 'singular noun, the currently active theme'
 							} ) }</span>
 						}
-						{ price && ! purchased &&
-							<span className="theme-badge__price">{ price }</span>
+						{
+							config.isEnabled( 'jetpack/pijp' )
+							? ! this.props.isJetpack && price && ! purchased &&
```

Comment:
For this first release, we won't be making any further UX tweaks in this PR. That can be moved to a separate issue once this is merged.
cc @dereksmart